src-java/kilda-persistence-tinkerpop/src/main/java/org/openkilda/persistence/ferma/frames/FlowMirrorPathFrame.java

```diff
@@ -0,0 +1,229 @@
+import org.apache.tinkerpop.gremlin.structure.Direction;
+import org.apache.tinkerpop.gremlin.structure.Edge;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+public abstract class FlowMirrorPathFrame extends KildaBaseVertexFrame implements FlowMirrorPathData {
+    public static final String FRAME_LABEL = "flow_mirror_path";
+    public static final String OWNS_SEGMENTS_EDGE = "owns";
+    public static final String PATH_ID_PROPERTY = "path_id";
+    public static final String SRC_SWITCH_ID_PROPERTY = "src_switch_id";
+    public static final String DST_SWITCH_ID_PROPERTY = "dst_switch_id";
+    public static final String DST_PORT_PROPERTY = "dst_port";
+    public static final String DST_VLAN_PROPERTY = "dst_vlan";
+    public static final String COOKIE_PROPERTY = "cookie";
+    public static final String IGNORE_BANDWIDTH_PROPERTY = "ignore_bandwidth";
+    public static final String BANDWIDTH_PROPERTY = "bandwidth";
+    public static final String DST_MULTI_TABLE_PROPERTY = "dst_with_multi_table";
+
+    private Switch srcSwitch;
+    private Switch destSwitch;
+    private FlowMirrorPoints flowMirrorPoints;
+    private List<PathSegment> segments;
+
+    @Override
+    @Property(PATH_ID_PROPERTY)
+    @Convert(PathIdConverter.class)
+    public abstract PathId getPathId();
+
+    @Override
+    @Property(PATH_ID_PROPERTY)
+    @Convert(PathIdConverter.class)
+    public abstract void setPathId(@NonNull PathId pathId);
+
+    @Override
+    @Property(SRC_SWITCH_ID_PROPERTY)
+    @Convert(SwitchIdConverter.class)
+    public abstract SwitchId getSrcSwitchId();
+
+    @Override
+    @Property(DST_SWITCH_ID_PROPERTY)
+    @Convert(SwitchIdConverter.class)
+    public abstract SwitchId getDestSwitchId();
+
+    @Override
+    @Property(DST_PORT_PROPERTY)
+    public abstract int getDestPort();
+
+    @Override
+    @Property(DST_PORT_PROPERTY)
+    public abstract void setDestPort(int destPort);
+
+    @Override
+    @Property(DST_VLAN_PROPERTY)
+    public abstract int getDestVlan();
+
+    @Override
+    @Property(DST_VLAN_PROPERTY)
+    public abstract void setDestVlan(int destVlan);
+
+    @Override
+    @Property(COOKIE_PROPERTY)
+    @Convert(FlowSegmentCookieConverter.class)
+    public abstract FlowSegmentCookie getCookie();
+
+    @Override
+    @Property(COOKIE_PROPERTY)
+    @Convert(FlowSegmentCookieConverter.class)
+    public abstract void setCookie(FlowSegmentCookie cookie);
+
+    @Override
+    @Property(BANDWIDTH_PROPERTY)
+    public abstract long getBandwidth();
+
+    @Override
+    @Property(BANDWIDTH_PROPERTY)
+    public abstract void setBandwidth(long bandwidth);
+
+    @Override
+    @Property(IGNORE_BANDWIDTH_PROPERTY)
+    public abstract boolean isIgnoreBandwidth();
+
+    @Override
+    @Property(IGNORE_BANDWIDTH_PROPERTY)
+    public abstract void setIgnoreBandwidth(boolean ignoreBandwidth);
+
+    @Override
+    @Property("status")
```

Comment:
why this property name is not const?

Reply:
We don't use the name of this property anywhere except for this place.